prime-router/metadata/schemas/covid-19.schema

```diff
@@ -776,7 +776,7 @@ elements:
 
   - name: processing_mode_code
     type: CODE
-    default: D
+    default: P
     cardinality: ZERO_OR_ONE
```

Comment:
So the default in covid-19.schema for processing_mode_code is now 'P' not 'D'.       This enables our unit/quicktest/and TestReportStream tests to pass; otherwise the new default quality filter setting just kills all our data.

I don't know why it was set to 'D', which seems dangerous to me anyway - we might mark real data as Debug data in production.